backend/src/app.ts

```diff
@@ -98,6 +100,7 @@ export function start(): Promise<Http2Server | undefined> {
   if (eventsEnabled) {
     startWatching()
   }
+  startAggregating()
```

Comment:
Since aggregating depends on watched resources, this should probably be moved inside the if block. `eventsEnabled` is true for the MCE pods, and false for the ACM pods.